src/makeInfinityAdder.js

```diff
@@ -4,7 +4,23 @@
  * @return {function}
  */
 function makeInfinityAdder() {
-  // write code here
+  let currentSum = 0;
+
+  const adder = function (num) {
+    if (num === undefined) {
```

Comment:
This num === undefined check doesn't work for a function that can take many arguments.
Here it is necessary to check whether the arguments are passed at all.

Example: 
```suggestion
    if (args.length === 0) {
```